crawl-ref/source/webserver/server.py

```diff
@@ -121,7 +121,7 @@ def bind_server():
             (r"/", MainHandler),
             (r"/socket", CrawlWebSocket),
             (r"/gamedata/(.*)/(.*)", GameDataHandler)
-            ], gzip=True, **settings)
+            ], gzip=use_gzip, **settings)
```

Comment:
This seems reasonable to me to have this as an option (though you may be underestimating how big some of the map json is), but as-is it'll run into the usual problems that are involved in adding a new value to `config.py`, because servers won't get the `config.py` part of this change. Should use `getattr(config, "use_gzip", True)` or something similar, cf. 84a5ed292d.